tests/Helpers/DateIntervalFormatter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Multitenancy\Helpers;
+
+/**
+ * @coversNothing
+ *
+ * @SuppressWarnings(PHPMD.CyclomaticComplexity) Complexity required to format DateInterval
+ * @SuppressWarnings(PHPMD.NPathComplexity) Complexity required to format DateInterval
+ */
+final class DateIntervalFormatter
```

Comment:
Rashmit couldnt find anywhere this class is used. If you're not using it, lets get rid of it